src/client/shared/ui/ui-button/styles/button-variants.scss

```
@@ -0,0 +1,48 @@
+  'secondary':   (var(--secondary),   var(--secondary-foreground)),
+  'destructive': (var(--destructive), white),
+  'muted':       (var(--muted),       var(--muted-foreground)),
+  'accent':      (var(--accent),      var(--accent-foreground)),
+);
+
+$sizes: (
+  'default': (2.25rem, 1rem,   0.5rem),
+  'xs':      (1.5rem,  0.5rem, 0),
+  'sm':      (2rem,    0.75rem, 0),
+  'lg':      (2.5rem,  1.5rem, 0),
+);
```

Stylelint (17.5.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)